asgs_main.sh

```diff
@@ -1333,9 +1333,9 @@ submitJob()
    echo "hpc.job.${JOBTYPE}.cloptions : \"$CLOPTIONS\"" >> $ADVISDIR/$ENSTORM/run.properties
    echo "hpc.job.${JOBTYPE}.localhotstart : $LOCALHOTSTART" >> $ADVISDIR/$ENSTORM/run.properties
    _CPUREQUEST=$(($NCPU + $NUMWRITERS))
-   _PPN=$(HPC_PPN_Hint "parallel" "$QUEUENAME" "$HPCENV" "$QOS" "$PPN" )
-   _RESERVATION=$(HPC_Reservation_Hint "$RESERVATION" "$HPCENV" "$QOS" "$CPUREQUEST")
    _QUEUENAME=$(HPC_Queue_Hint "$QUEUENAME" "$HPCENV" "$QOS" "$CPUREQUEST")
+   _PPN=$(HPC_PPN_Hint "parallel" "$QUEUENAME" "$HPCENV" "$QOS" "$PPN" "$CPUREQUEST")
+   _RESERVATION=$(HPC_Reservation_Hint "$RESERVATION" "$HPCENV" "$QOS" "$CPUREQUEST")
```

Comment:
Looks like you found and fixed an issue with this call to `HPC_Reservation_Hint`, too. Good catch. We should just be aware of anything that might have been relying on this bug (no reason to not approve). Still a good catch and fix.